Message class tests

Gal-f opened the pull request:
Testing all class methods of Message class.
Also moving one line from Category tests to Message tests in order
to delete the test category only after all message tests are completed.

Resolves #45 alongside PR #78

Gal-f committed:
Adding tests for Mesages class methods

Testing all class methods of Message class.
Also moving one line from Category tests to Message tests in order
to delete the test category only after all message tests are completed.

simzacks reviewed (changes requested):
The file has conflicts. please resolve before commiting

Gal-f committed:
Added global variable for testing + some fixes

Added a global variable of class Message in order to add, edit and
delete it in different test methods.

Also fixed a few convention errors.

ravidbro commented:
The tests are failing

Gal-f committed:
Removing 'categories' parameter from 'addMessage'

This parameter will probably be back in the future after adjusting
the method in 'models.py', in order to allow further functionality.
At this point, it only breaks the code from running so it is removed.

Gal-f committed:
Removing 'categories' parameter from 'addMessage'

This parameter will probably be back in the future after adjusting
the method in 'models.py', in order to allow further functionality.
At this point, it only breaks the code from running so it is removed.
